src/Support/Str.php

```diff
@@ -85,7 +85,7 @@ public static function stripTags($html, $tags_list = [])
 
     public static function slug($string, $separator = '-', $language = 'en')
     {
-        // Statamic is a-OK with underscores in slugs.
+        // Statamic is OK with underscores in slugs.
```

Comment:
a-OK is a-OK. 😉 

https://www.merriam-webster.com/dictionary/A-OK

Reply:
A-OK :-) Non-native english speaker. Thanks for the hint.

Reply:
> If we're allowing a . at all, why only .html? Thats what the documentation says. What else would you use as an extension? Maybe `.htm` or `.aspx`?

Reply:
I mean maybe you want a dot in there that isn't a file extension. The route is just a fake file extension anyway - we're not actually serving a static html file, it's just there for people who don't want to redirect old URLs.

`an-article-about-robots.txt-files`,
`version-1.2-has-been-released`,
etc